lib/blockscore/question_set.rb

```diff
@@ -16,7 +16,7 @@ def score(answers = nil)
     private
 
     def rescore(answers)
-      @attributes = post("#{endpoint}/#{id}/score", answers: answers).attributes
+      capture_attributes(post("#{endpoint}/#{id}/score", answers: answers))
```

Comment:
I wounder if it might be possible to use a `Pathname` instance to build the path from.

`endpoint.join(id, 'score')` would read much nicer than the string interpolation. Also Pathname provides a much more narrow interface than a `String` instance.